src/builtin/tables.scrbl

```diff
@@ -217,152 +345,294 @@ check:
-    { {sum + n; count + 1}; (sum + n) / (count + 1) }
-  end
-}
+The simplest examples of reducing use reducers built into Pyret.
+
+In each reducer below, you will need to specify
+a name for the new column and which existing column new value
+will be based on.  
+
+@margin-note{To use @pyret{extend} expressions you will need to
+@pyret{import} or @pyret{include} @pyret{tables}.}
+
+@value["running-sum" (Red-of N N N)]
+
+Creates a new column where in each row,
+the running sum will be the added
+value of the cell in the selected column plus all the cells
+@italic{above} the cell in the same column.
+
+@examples{
+import tables as T
+dem-primary-delegates = table: state :: String, clinton :: Number, 
+  sanders :: Number
+  row: "Iowa", 29, 21
+  row: "New Hampshire", 15, 16
+  row: "Nevada", 27, 16
+  row: "South Carolina", 44, 14
+end
+running-total-delegates = extend dem-primary-delegates 
+  using clinton, sanders:
+  total-clinton: T.running-sum of clinton,
+  total-sanders: T.running-sum of sanders
+end
+print(running-total-delegates)
 }
 
-@subsection{Reducers}
+@(image "src/builtin/primaries.png")
 
-The following reducers are provided:
+@value["difference" (Red-of N N N)]{
 
-@value["running-mean" (Red-of N N N)]
+The @pyret{difference} extender creates a new column 
+containing the difference between the value in
+the current row (of the selected column) minus the value in @italic{only}
+the row directly above.  In the first row, zero is subtracted from the value.
+
+@margin-note{Both @pyret{difference} and @pyret{difference-from} do
+@italic{not} calculate a running difference, only the difference between
+the selected row and the single row above.}
+
+@examples{
+import tables as T
+test-scores = table: year :: Number, 
+  math-score :: Number, reading-score :: Number
+  row: 2014, 87, 89
+  row: 2015, 98, 93
+  row: 2016, 79, 83
+  row: 2017, 85, 90
+end
+changes-by-year = extend test-scores using math-score, reading-score:
+  math-change-from-previous: T.difference of math-score,
+  reading-change-from-previous: T.difference of reading-score
+end
+}
+@(image "src/builtin/difference-table.png")                           
+}
 
 @function["difference-from"
   #:contract (a-arrow N (Red-of N N N))
   #:args '(("start-value" #f))
   #:return (Red-of N N N)]{
 
-  }
-@function["running-fold"
-  #:contract (a-arrow "Result" (a-arrow "Result" "Col" "Result") (Red-of "Result" "Col" "Result"))
-  #:args '(("start-value" #f) ("combiner" #f))
-  #:return (Red-of "Result" "Col" "Result")]{
+Like @pyret{difference}, except the starting value is specified, instead
+of defaulting to 0.
 
-@pyret-block{
+@pyret-block[#:style "bad-ex"]{
+# This DOES NOT calculate a running balance in a checking account.
+checks = table: check-number :: Number, withdrawal :: Number
+  row: 001, 50
+  row: 002, 100
+  row: 003, 500
+end
+with-checking-balance = extend checks using withdrawal:
+  current-balance: T.difference-from(1000) of withdrawal
```

Comment:
Yeah, it might be too confusing as a negative example.  I'm not sure how to clearly illustrate an incorrect result.  I felt like including it because it was actually my first attempt at using difference-from, and I figured if I'd made that mistake, someone else might.  Not married to it though!